lab-matt/cowsay.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const http = require('http');
+const parseUrl = require('url').parse;
+const parseQuery = require('querystring').parse;
+const cowsay = require('cowsay');
+
+const parseBody = require('./lib/parse-json');
+
+const port = process.argv[2] || 5000;
+
+const server = http.createServer(function(req, res){
+  req.url = parseUrl(req.url);
+  req.url.query = parseQuery(req.url.query);
+
+  if(req.method === 'POST' || 'PUT' || 'DELETE'){
```

Comment:
This pattern for determining path routes is okay, I would consider putting the req.method check inside a route declaration to avoid repeating yourself.